python/py_vapid/__init__.py

```diff
@@ -88,20 +102,21 @@ def validate(self, token):
         token = base64.urlsafe_b64encode(sig)
         return token
 
-    def verifyToken(self, sig, token):
+    def verify_token(self, sig, token):
+        """Verify the signature against the token."""
         hsig = base64.urlsafe_b64decode(sig)
         return self.public_key.verify(hsig, token,
                                       hashfunc=self._hasher)
 
     def sign(self, claims, crypto_key=None):
         """Sign a set of claims.
-
         :param claims: JSON object containing the JWT claims to use.
         :param crypto_key: Optional existing crypto_key header content. The
             vapid public key will be appended to this data.
         :returns result: a hash containing the header fields to use in
             the subscription update.
         """
+        from jose import jws
```

Comment:
Nope, my mistake. Was originally using this without the whole library and forgot to remove it before submitting changes.